docs/content/en/usage.md

```diff
@@ -55,3 +62,20 @@ Use `LazyNuxtDynamic` if you want the component itself being imported lazily.
   <LazyNuxtDynamic component="Logo" />
 </template>
 ```
+
+
+## Assign Ref to child component
+
+Use `componentRef` to assign a `ref` to the child component.
+
+```vue
+<template>
+  <NuxtDynamic component="Logo" componentRef="logoChild" ref="logo" />
+</template>
+```
+
+This allows you to call methods on the child component or access its data:
+
+```js
+this.$refs.logo.[0].$refs.logoChild
```

Comment:
correct me if I'm wrong but I believe `this.$refs.logo` contains a reference to the component instead of an array - can you confirm that?

Reply:
First Comment:
You are completely right. XD.

Second comment:
You are right here as well. I scrambled it up a bit because I had a very special use case and used it with a for loop. Than `this.$refs` is an array as mentioned here: https://vuejs.org/v2/api/#ref

The correct usage is this: `this.$refs.logo.$refs.logoChild` Pretty simple and straight forward. Just tried it. I am going to update the docs.